test/stdlib/StringDescribing.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+private class Foo {}
+class Bar {}
+
+var StringDescribingTestSuite = TestSuite("StringDescribing")
+
+StringDescribingTestSuite.test("String(describing:) shouldn't include extra stuff if the class is private") {
+  expectEqual(String(describing: Foo.self), "Foo")
+  expectEqual(String(describing: Bar.self), "Bar")
+}
+
+StringDescribingTestSuite.test("String(reflecting:) should include extra stuff if the class is private") {
+  expectEqual(String(reflecting: Bar.self), "main.Bar")
+  expectNotEqual(String(reflecting: Foo.self), "main.Foo")
```

Comment:
Ah, sorry, I'd like you to put the actual thing it uses here. It's not that we care about it directly, but that we'd want to know if any part of it disappeared. (That is, "Foo" and "main.(private).Foo" would also pass this test.)